playground/nextjs/src/posthog.ts

```diff
@@ -57,7 +60,7 @@ if (typeof window !== 'undefined') {
         persistence_name: `${process.env.NEXT_PUBLIC_POSTHOG_KEY}_nextjs`,
         ...configForConsent(),
     })
-
+    ;(window as any).posthog = posthog
```

Comment:
Is this still needed for anything? We should probably remove if not, as site apps/destinations should work without it.

Reply:
No, that was just for debugging. Removed!